jip/util.py

```diff
@@ -71,6 +71,7 @@ def download_string(url):
     import requests
     try:
         response = requests.get(url, headers={ 'User-Agent': JIP_USER_AGENT})
+        response.encoding = 'utf-8'
```

Comment:
Yes, it's a dubious operation, but Maven repository gives UTF-8 encoded files without content encoding header, so the 'requests' incorrectly assigns encoding 'iso-8859-1'.